src/software/simulation/physics/physics_robot.h

```diff
@@ -250,17 +244,17 @@ class PhysicsRobot
 
     // See https://box2d.org/manual.pdf chapters 6 and 7 more information on Shapes,
     // Bodies, and Fixtures
-    b2Body* robot_body;
+    b2Body *robot_body;
 
-    RobotId robot_id;
+    unsigned int robot_id;
```

Comment:
Any particular reason we made this type change?

Reply:
I stopped including `robot.h` which has the typedef